client/src/components/Navbar.jsx

```diff
@@ -109,7 +111,12 @@ export function Navbar(props) {
           </Button>
         )}
         {isLoggedIn && (
-          <div>
+          <div style={{display:'flex', alignItems: 'center'}}>
+            {userData.profile_pic ? <Avatar
+              alt="User picture"
+              src={`/images/${userData.profile_pic}`}
+              sx={{ width: 35, height:35, marginRight: '15px' }}
+            /> : <Avatar alt="User picture" {...navBarAvatar(`${userData.first_name} ${userData.last_name}`)}/> }
```

Comment:
Minor, but might be cleaner to pass the initials in here instead of extracting them in the Avatar component itself.

Reply:
I am also passing in a bunch of styling as part of the nav Bar Avatar so i prefer to leave it extracted in the styling.js for now.